docs/source/index.rst

```diff
@@ -1,8 +1,10 @@
+.. page-index:
```

Comment:
Do you know what these GitHub Actions mean?

Reply:
I would think GitHub is clever enough to link warnings/errors to the code. This is probably one... Of many. I only cleaned up ~30% of the errors. There are still so many left. Especially from absail.

Reply:
Yeah abseil, wasn't a great idea....

Reply:
Well. It's a start. Did you see the argparse flags that Pytorch Lightning introduces? I really like that approach